line-sdk/build.gradle

```diff
@@ -8,8 +8,8 @@ android {
     defaultConfig {
         minSdkVersion 17
         targetSdkVersion 27
-        versionCode 50000
-        versionName "5.0.1"
+        versionCode 50010
```

Comment:
sorry, I am not sure about the format of `versionCode`,
for `versionName=5.1`, should `versionCode` be `51000`  or `50010`?

Reply:
For version code, there's no specific rules for it yet, as long as it's increased in each version. I think diff 10 is enough if there should be any minor release for 5.1. That's why I change it from 5000 to 50010 for v5.1 .

Reply:
Understand